frontend/packages/pipelines-plugin/integration-tests/features/pipelines/pipeline-metrics.feature

```diff
@@ -25,13 +25,13 @@ Feature: Pipeline metrics
               And user can see Pipeline success ratio, Number of Pipeline Runs, Pipeline Run duration, Task Run duration graphs
 
 
-        @regression @to-do
+        @regression
         Scenario: No datapoint graphs in metrics tab: P-04-TC03
             Given pipeline "pipeline-metrics-two" is present on Pipeline Details page
-             When user clicks on pipeline "pipeline-metrics-two"
               And user selects option "Start" from Actions menu drop down
```

Comment:
since you removed the When clause, please add it to the next line, so we keep the given-when-then format